ironfish-cli/src/commands/wallet/burn.ts

```diff
@@ -46,9 +46,9 @@ export class Burn extends IronfishCommand {
       minimum: 1n,
       flagName: 'fee rate',
     }),
-    amount: IronFlag({
+    amount: ValueFlag({
```

Comment:
Should handle parsing decimal places here and in other commands. We prob have to keep this as a string flag and parse it later